setup_creators/solutions/scripts/plot-positions.py

```
@@ -0,0 +1,28 @@
+
+# compute variant density in each window
+h, _ = np.histogram(pos, bins=bins)
+y = h / window_size
+
+# plot
+fig, ax = plt.subplots(figsize=(12, 3))
+sns.despine(ax=ax)  # , offset=10)
+ax.plot(x, y)
+ax.set_xlabel("Chromosome position (bp)")
+ax.set_ylabel("Variant density (bp$^{-1}$)")
+plt.savefig(snakemake.output[0])
```

Ruff

Undefined name `snakemake`

(F821)